axum/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - **added:** `middleware::from_fn` for creating middleware from async functions.
   This previously lived in axum-extra but has been moved to axum ([#719])
+- **added:** Implement `IntoResponse` for `bytes::BytesMut` and `bytes::Chain<T,U>` ([#767])
```

Comment:
```suggestion
- **added:** Implement `IntoResponse` for `bytes::BytesMut` and `bytes::Chain<T, U>` ([#767])
```

Also have to add a reference for `[#767]` otherwise the link wont work.